www/__mocks__/timelineHelperMocks.ts

```diff
@@ -0,0 +1,254 @@
+import { MetaData, ServerData, ServerResponse } from '../js/types/serverData';
+import {
+  CompositeTrip,
+  ConfirmedPlace,
+  FilteredLocation,
+  TripTransition,
+  UnprocessedTrip,
+} from '../js/types/diaryTypes';
+import { LabelOptions } from '../js/types/labelTypes';
+
+const mockMetaData: MetaData = {
+  write_ts: 1,
+  key: 'test/value/one',
+  platform: 'test',
+  time_zone: 'America/Los_Angeles',
+  write_fmt_time: '1969-07-16T07:01:49.000Z',
+  write_local_dt: null,
+  origin_key: '1',
+};
+
+export const mockLabelOptions: LabelOptions = {
+  MODE: null,
+  PURPOSE: null,
+  REPLACED_MODE: null,
+  translations: null,
+};
+
+const mockConfirmedPlaceData: ConfirmedPlace = {
+  source: 'DwellSegmentationTimeFilter',
+  location: {
+    type: 'Point',
+    coordinates: [-122.0876886, 37.3887767],
+  },
+  cleaned_place: {
+    $oid: '6553c3a0f27f16fbf9d1def1',
+  },
+  additions: [],
+  user_input: {},
+  enter_fmt_time: '2015-07-22T08:14:53.881000-07:00',
+  exit_fmt_time: '2015-07-22T08:14:53.881000-07:00',
+  starting_trip: {
+    $oid: '6553c3a1f27f16fbf9d1df15',
+  },
+  ending_trip: {
+    $oid: '6553c3a1f27f16fbf9d1df15',
+  },
```

Comment:
Do you need all this information? The oids will be different for every run, so I am not sure you can check against it. And if you can't check against it, do you still need it?